src/mon/Monitor.cc

```diff
@@ -3762,14 +3762,14 @@ void Monitor::handle_ping(MonOpRequestRef op)
   MPing *reply = new MPing;
   entity_inst_t inst = m->get_source_inst();
   bufferlist payload;
-  Formatter *f = new JSONFormatter(true);
```

Comment:
@tchaikov Sure. http://tracker.ceph.com/issues/15793
But I doubt I shall update the commit message correspondingly, since yuri is already testing this, right?